cellprofiler/modules/exporttocellh5.py

```diff
@@ -306,21 +310,19 @@ def handle_interaction(self, master_file, pid, path):
         master_dict = self.get_dictionary().setdefault(master_file, {})
         if pid not in master_dict:
             md_head, md_tail = os.path.splitext(master_file)
-            subfile = "%s_%s%s" % (md_head, str(pid), md_tail)
```

Comment:
Yes. One file per worker should be the goal. So far, several channels can be written by a worker. The next would be to add temporal support, and - if available - write a movie. How is it with z-slices? 

In case, there is no meta data (for plate, well, site), what should I use? Cellh5 requires this information as a key for the image... I could just take Constant_place holders plus an auto-incrementing digit!?

If only site is missing, this should default to sth constant "1" or "0"..., I suppose